app/helpers/application_helper.rb

```diff
@@ -171,15 +167,15 @@ def non_anonymous_text_byline(creation)
   end
 
   def pseud_text(pseud)
-      pseud.byline
+    pseud.byline
   end
 
-   def link_to_modal(content="",options = {})   
+   def link_to_modal(content="",options = {})
```

Comment:
Inconsistent indentation detected.<br>Space missing after comma.<br>Surrounding space missing in default value assignment.